### PR TITLE
Documentation review

### PR DESCRIPTION
**Pull request by louperelo**
Updates on the output.md, usage.md and base.config (added BAKTA processes). There are still some TODO statements in the base.config to be addressed (or removed) before release.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ x] Usage Documentation in `docs/usage.md` is updated.
- [ x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -333,17 +333,20 @@ We have standardised the structure of Nextflow DSL2 pipelines such that all modu
...
 process {
     withName: 'NFCORE_RNASEQ:RNASEQ:ALIGN_STAR:STAR_ALIGN' {
         memory = 100.GB
+        time = 8.h
+        cpus = 6
     }
 }
 ```
 
+
```
Comment (jfy133) [inline docs/usage.md:349]: I would recommend NOT changing these - they come from the template, and these resource conficugations are better documentd on the nf-core website

File: conf/base.config [hunk 2]
```diff
@@ -97,6 +98,16 @@ process {
...
 
+    withName: BAKTA_BAKTADBDOWNLOAD {
+        memory = { check_max( 4.GB * task.attempt, 'memory'  ) }
+        cpus   = { check_max( 4    * task.attempt, 'cpus'   ) }
+    }
+
+    withName: BAKTA_BAKTA {
+        memory = { check_max( 4.GB * task.attempt, 'memory'  ) }
+        cpus   = { check_max( 4    * task.attempt, 'cpus'   ) }
+    }
```
Comment (jfy133) [inline conf/base.config:109]: Why does generic `process_single` or `process_low` labels not work here?

**Review by jfy133 — COMMENTED** [review]
Overall OK, not sure about the rsource changs for BKATa though?

**Review by jfy133 — APPROVED** [review]

**Comment by louperelo** [comment]
@nf-core-bot fix linting

**Pull request merged**